app/src/lib/stores/helpers/pull-request-updater.ts

```diff
@@ -104,9 +106,14 @@ export class PullRequestUpdater {
     this.timeoutHandles.set(TimeoutHandles.PushedPullRequest, handle)
   }
 
-  private async refreshPullRequestStatus(repository: GitHubRepository) {
-    await this.store.refreshPullRequestStatuses(this.repository, this.account)
-    const prs = await this.store.getPullRequests(repository)
+  private async refreshPullRequestStatus(repository: Repository) {
```

Comment:
We already track `this.repository` from the constructor - we can eliminate this parameter for a bit of :lipstick: